### ISSUE TITLE
Why use the tf-nightly?

### ISSUE THREAD

**Issue by gomes-eric** [tensorflow/agents#237]
Hi,

Could someone answer me why we should use tf-nightly instead of tensorflow since tf-nightly is conducive to unexpected failures?

Thanks in advance!

**Comment by christianhidber** [comment]
maybe this helps, although it does not answer your question: 
on 12/10/19 a new rc version was released (pip install tf-agents)
instead of tf-nightly we are now using this one.

**Comment by tfboyd** [comment]
Confirming and maybe expanding on @christianhidber answer.  

The git repo is updated constantly from our internal head, which means it can drift and not work with the released version of TensorFlow.  We have nightly tests that run against the nightly tensorflow and the head of tf-agents.  It is 100% possible for it not to work but it usually is fine.

If you want something more stable we took a snap shot back in December and called it 0.3.0.  I executed the tests against TF 2.0 and TF 1.15.  There is both a branch and a pip release.  I am pretty sure it broke with TF 2.1.0 due to some newly deprecated code in TensorFlow 2.1.0.  

https://github.com/tensorflow/agents#releases

I hope to do more releases, and improve the process, based on demand.
